system-design/youtube-start-seconds.md

```diff
@@ -0,0 +1,58 @@
+# YouTube Iframe API - startSecond
+
+The property **startSeconds** lets you skip the given seconds from the start of video.
+
+## Brief Overview
+
+- **startSeconds** load video and skip the given seconds from the start.
```

Comment:
Does `startSeconds` only accepts values in seconds?

Is `startSecond` inclusive or exclusive (Does the player seeks till the _start_ of `startSeconds` value or the _end_?).

Reply:
Yes its accept value in seconds and is inclusive updated the overview accordingly.